<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员登录 - 数据库管理系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>管理员账号:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userNameEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>密      码:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passWordEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="loginButton">
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
          <property name="orientation">
              <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
              <size>
                  <width>20</width>
                  <height>40</height>
              </size>
          </property>
      </spacer>
     </item>
        <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                    <spacer name="horizontalSpacer_3">
                        <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>40</width>
                                <height>20</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item>
                    <widget class="QPushButton" name="backButton">
                        <property name="text">
                            <string>返回</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
